psalm-baseline.xml

```diff
@@ -797,27 +779,22 @@
     </MissingReturnType>
   </file>
   <file src="src/Generator/TypeGenerator.php">
-    <InvalidScalarArgument occurrences="4">
-      <code>$trimmedNullable</code>
-      <code>$trimmedType</code>
-      <code>$trimmedType</code>
-      <code>$trimmedType</code>
-    </InvalidScalarArgument>
-    <MissingReturnType occurrences="1">
-      <code>generate</code>
-    </MissingReturnType>
-    <MixedArgument occurrences="1">
-      <code>$this-&gt;generate()</code>
-    </MixedArgument>
-    <PossiblyInvalidPropertyAssignmentValue occurrences="2">
-      <code>$nullable</code>
-      <code>$trimmedType</code>
-    </PossiblyInvalidPropertyAssignmentValue>
-    <PropertyNotSetInConstructor occurrences="3">
-      <code>$isInternalPhpType</code>
-      <code>$nullable</code>
-      <code>$type</code>
-    </PropertyNotSetInConstructor>
+    <ImpureMethodCall occurrences="7">
+      <code>allowsNull</code>
+      <code>getName</code>
+      <code>getName</code>
+      <code>getName</code>
+      <code>getName</code>
+      <code>getParentClass</code>
+      <code>getTypes</code>
+    </ImpureMethodCall>
+    <MixedArgument occurrences="1"/>
+  </file>
+  <file src="src/Generator/TypeGenerator/AtomicType.php">
+    <LessSpecificReturnStatement occurrences="1"/>
+    <MoreSpecificReturnType occurrences="1">
+      <code>non-empty-string</code>
```

Comment:
At this code location, the type inferred by psalm is `string` instead of `non-empty-string`, even though all code paths lead to non-empty strings being produced.

Potentially upstream issue to be fixed, but not a problem in our codebase.